plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/panel/ProcessesPanelPresenter.java

```diff
@@ -364,12 +367,9 @@ public void onTreeNodeSelected(ProcessTreeNode node) {
      *         machine to retrieve address
      * @return ssh service address in format host:port
      */
-    private String getSshServerAddress(MachineDto machine) {
-        if (machine.getRuntime().getServers().containsKey(SSH_PORT + "/tcp")) {
-            return machine.getRuntime().getServers().get(SSH_PORT + "/tcp").getAddress();
-        } else {
-            return null;
-        }
+    private String getSshServerAddress(Machine machine) {
+        Map<String, ? extends Server> servers = machine.getRuntime().getServers();
+        return servers.containsKey(SSH_PORT + "/tcp") ? servers.get(SSH_PORT + "/tcp").getAddress() : null;
```

Comment:
Looks like we can simplify this complex construction here by decomposing it into `servers.get(SSH_PORT + "/tcp")` variable and if/else block
```java
final Server sshServer = servers.get(SSH_PORT + "/tcp");
if (sshServer == null){
   return null;
} else {
   return sshServer.getAddress();
}
```
or something similar. What do you think, does that fits here in your opinion?